spec/spree_shared_spec.rb

```diff
@@ -21,6 +21,7 @@
     end
     
     Apartment::Tenant.switch!
+    Spree::Preferences::Store.instance.persistence = true
```

Comment:
Dowiedziałeś się w końcu dlaczego to tutaj musi być? Bo to jest mega nieintuicyjne imo